content/blogs/release-0-23.md

```diff
@@ -0,0 +1,442 @@
+</div>
+
+---
+
+Let's dive into these highlights and other enhancements in more detail.
+
+## Feature Highlights
+
+### Multi Panel Editor
+
+The Multi Panel Editor transforms how you design and manage workflows by introducing a unified interface with split-screen capabilities. This powerful enhancement allows you to simultaneously view and edit your workflows in multiple formats - YAML code, visual no-code editor, topology view, and documentation - all within a single, integrated workspace.
+
+With this flexible interface, you can:
+- Edit YAML directly while previewing the visual representation in real-time
+- Switch between code and no-code approaches based on your preference or task complexity
+- Visualize task dependencies through the topology view while making edits
+- Reference documentation without leaving your editing environment
+
+You can customize your experience by opening only the panels you need, creating a workspace tailored to your workflow development style. This enhancement significantly improves productivity by reducing context switching and providing multiple perspectives on your workflows simultaneously.
+
+<div class="video-container">
+  <iframe src="https://www.youtube.com/embed/PLACEHOLDER" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+</div>
+
+### Unit Test Flows
+
+As workflows grow in complexity, so does the need to test them reliably. Kestra introduces native support for Unit Testing, giving users powerful new capabilities to validate their flows with confidence and precision.
+Until now, users relied on manual execution or external scripting to verify behavior. With Unit Test support, you can now define expected outcomes, isolate tasks, and detect regressions early—directly inside your development environment.
+
+Key components of the Unit Testing include:
+
+- **Test Suites**: Create test suites targeting individual tasks or full sections of your flows. Each suite encapsulates a set of test cases, allowing you to cover multiple paths and edge cases without executing the full logic end-to-end.
+- **Fixtures**: Define fixtures to simulate inputs or task outputs. This skips real execution for mocked tasks, making your tests faster, deterministic, and easier to isolate.
+- **Assertions**: Each test can include assertions to check for expected outcomes—task results, flow variables, execution paths, or any output produced by the flow. This helps ensure your flow behaves correctly under different conditions.
+- **API Access**: You can call the Unit Test programmatically via Kestra API, enabling automation in CI/CD pipelines, custom tooling, or integration with development workflows.
```

Comment:
afaik this is not available yet -- @AcevedoR can you confirm? for now, I'll open an issue for 0.24 👍

Reply:
https://github.com/kestra-io/kestra-ee/issues/3826

Reply:
a simple API is available but not documented